test/tstDetailsBufferOptimization.cpp

```diff
@@ -11,10 +11,9 @@
 
 #include "ArborX_EnableDeviceTypes.hpp" // ARBORX_DEVICE_TYPES
 #include "ArborX_EnableViewComparison.hpp"
-#include <ArborX_DetailsBufferOptimization.hpp>
-//#include <ArborX_Callbacks.hpp>
-#include <ArborX_DetailsBoundingVolumeHierarchyImpl.hpp> // FIXME
+#include <ArborX_DetailsCrsGraphWrapperImpl.hpp>
```

Comment:
You probably should update the test file name and test module name below

Reply:
Yeah, I thought about it, but forgot to do it.